src/tests/e2e/pages/disconnect.page.js

```diff
@@ -1,12 +1,30 @@
+import MetamaskPage from './metamask.page'
 import log from 'loglevel'
 log.setDefaultLevel('info')
 
 class DisconnectPage {
+  constructor () {
+    this.metamaskPage = new MetamaskPage()
```

Comment:
This should be declared when MetaMask is needed.

Reply:
fixed